rust/CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+TODO organize this
+
+Every md file has a html relative symlink to index.html:
```

Comment:
Better to explain the reason? I first felt confused until I find out we are loading the md dynamically according to the url.